robocop/checkers/spacing.py

```diff
@@ -186,14 +186,14 @@ def verify_empty_lines_between_nodes(self, node, node_type, issue_name, allowed_
             if not isinstance(child, node_type):
                 continue
             empty_lines = self.verify_empty_lines(child)
-            if index < last_index and empty_lines != allowed_empty_lines:
+            if empty_lines != allowed_empty_lines and allowed_empty_lines != -1 and index < last_index:
```

Comment:
I think I will split those conditions into separate lines.

Reply:
I refactored it this way:
```
if allowed_empty_lines not in (empty_lines, -1) and index < last_index
```